<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2098</width>
    <height>1002</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>1621</width>
      <height>931</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Residual</string>
     </attribute>
     <widget class="QCustomPlot" name="CP_RESIDUAL1" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>770</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="CP_RESIDUAL2" native="true">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>20</y>
        <width>770</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="CP_RESIDUAL3" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>770</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="CP_RESIDUAL4" native="true">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>470</y>
        <width>770</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Custom</string>
     </attribute>
     <widget class="QCustomPlot" name="CP_CUSTOM2" native="true">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>20</y>
        <width>770</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="CP_CUSTOM3" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>770</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="CP_CUSTOM4" native="true">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>470</y>
        <width>770</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="CP_CUSTOM1" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>770</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_control">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>CONTROL</string>
    </property>
    <widget class="QPushButton" name="BT_START">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BT_SWING">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>SWING</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BT_CUSTOM1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>CUSTOM1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BT_CUSTOM2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>CUSTOM2</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2098</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
